.github/workflows/build.yml

```diff
@@ -10,6 +10,9 @@ on:
   pull_request:
     branches:
       - '*'
+  schedule:
+    # Trigger workflow run every workday (Mon-Fri) at 0700
+    - cron:  '0 7 * * 1,2,3,4,5'
```

Comment:
Might not hurt to have it running in the weekends as well. Only potentially two more messages in the slack channel, but then if someone works on the weekend the issue might get picked up and a fix can be ready to merge first thing on monday.

Reply:
> Might not hurt to have it running in the weekends as well. Only potentially two more messages in the slack channel, but then if someone works on the weekend the issue might get picked up and a fix can be ready to merge first thing on monday.

@pkhansen My thinking was actually to not disturb devs on weekends. The cron trigger runs early monday morning anyways which in case of failure can be picked as first thing in the morning to be fixed. But your reasoning makes sense as well.

Reply:
No, you're right. I guess there is certain implications by having something like this going on in the weekends.